### DISCUSSION TITLE
Basic Usage Question

### DISCUSSION THREAD

**Discussion by quartet4** [stepwisehq/prosemirror-dotnet#23] [Q&A] [answered]
I'm trying to run the basic usage code and am running into an error:

```
Unable to initialize 'ContentLike?' with a collection expression because the following required members will not be initialized:
* ContentLike.Content
```

![contentlikeerror](https://github.com/user-attachments/assets/6ccdf10b-72fd-4c96-b19f-f7f8d01e7ede)

It seems like it should work as is. Do you have any suggestions?

**Comment by ProTip** [comment]
Try `(Node)schema.Text("Hello World")` .

I have been iterating on the doc "DSL" trying to get it close to the TypeScript experience, and the most recent version uses custom operators and a collection expression builder. It looks like a single `TextNode` in the list isn't being target typed to `Node`, which is the type for the only builder signature ATM.

**Reply by quartet4** [reply, marked as answer]
I tried using Node but it was giving me the same error message. So then I tried it with Fragment and that seemed to do the trick. Thanks for pointing me in the right direction!

![image](https://github.com/user-attachments/assets/cd44eab7-0e15-4f5d-a1f2-a5e466011c00)

**Reply by ProTip** [reply]
I had some time to look into this today, but it turns out your first example actually works for me without error in VS 2022 and the latest .Net 8..
